public/404.html

```diff
@@ -5,37 +5,10 @@
     <meta charset='utf-8'>
     <meta content='hackathons, hackathon, hack-a-thon, programming, hardware, development, design, rit, ' name='keywords'>
     <meta content='width=device-width, initial-scale=1' name='viewport'>
-    <meta content='BrickHack: RIT’s premiere collegiate hackathon. April 18th ignites a weekend devoted to 400 designers and coders sinking 24 hours into building and creating.' name='description'>
-    <meta content='BrickHack' property='og:title'>
-    <meta content='http://brickhack.io' property='og:url'>
-    <meta content='BrickHack: RIT’s premiere collegiate hackathon. April 18th ignites a weekend devoted to 400 designers and coders sinking 24 hours into building and creating.' property='og:description'>
-    <meta content='http://brickhack.io/assets/brickhack_logo_v2_filled.png' property='og:image'>
-    <meta content='summary' name='twitter:card'>
-    <meta content='http://brickhak.io' name='twitter:url'>
-    <meta content='BrickHack' name='twitter:title'>
-    <meta content='/assets/brickhack_logo_v2_filled.png' name='twitter:image'>
-    <meta content='BrickHack: RIT’s premiere collegiate hackathon. April 18th ignites a weekend devoted to 400 designers and coders sinking 24 hours into building and creating.' name='twitter:description'>
-    <link href='/assets/favicon/apple-touch-icon-57x57.png' rel='apple-touch-icon' sizes='57x57'>
-    <link href='/assets/favicon/apple-touch-icon-114x114.png' rel='apple-touch-icon' sizes='114x114'>
-    <link href='/assets/favicon/apple-touch-icon-72x72.png' rel='apple-touch-icon' sizes='72x72'>
-    <link href='/assets/favicon/apple-touch-icon-144x144.png' rel='apple-touch-icon' sizes='144x144'>
-    <link href='/assets/favicon/apple-touch-icon-60x60.png' rel='apple-touch-icon' sizes='60x60'>
-    <link href='/assets/favicon/apple-touch-icon-120x120.png' rel='apple-touch-icon' sizes='120x120'>
-    <link href='/assets/favicon/apple-touch-icon-76x76.png' rel='apple-touch-icon' sizes='76x76'>
-    <link href='/assets/favicon/apple-touch-icon-152x152.png' rel='apple-touch-icon' sizes='152x152'>
-    <link href='/assets/favicon/apple-touch-icon-180x180.png' rel='apple-touch-icon' sizes='180x180'>
+    <meta content='BrickHack: RIT’s premiere collegiate hackathon. January 27th ignites a weekend devoted to 500+ designers and coders devoting 24 hours into building and creating.' name='description'>
+    <meta content='BrickHack' property='title'>
+    <meta content='http://brickhack.io' property='url'>
```

Comment:
I think we might as well remove these meta tags entirely? An error page shouldn't need to worry about SEO, plus its super easy for these to be forgotten about when the regular pages get updated